src/css/imports/mixins.less

```diff
@@ -345,6 +345,7 @@
           }
         }
 
+        .jw-text-alt,
```

Comment:
I thought it had at first but it actually looks okay there.  Will remove it and shift it downwards into an `.jw-ie` block.